horace_core/sqw/file_io/@faccess_sqw_v3/private/get_instr_or_sample_.m

```diff
@@ -19,10 +19,10 @@
 end
 res_block = get_all_instr_or_samples_(obj,field_name);
 if strcmp(field_name,'instrument') && isstruct(res_block) && numel(fields(res_block))==0
-    res_block  = IX_inst();
+    res_block  = IX_null_inst();
 end
 if strcmp(field_name,'sample') && isstruct(res_block) && numel(fields(res_block))==0
-    res_block  = IX_samp();
+    res_block  = IX_null_sample();
 end
```

Comment:
we never had `null_sample` -- `IX_samp` with lattice only have to be defined for anything to work.

Reply:
As previously noted, IX_samp is now conceptually an abstract class - the (Abstract) has only been removed to allow serializable to work. As we do not know what is in the sample at this point, IX_null_sample is the correct indicator for this.